sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.1.1 (Unreleased)
```

Comment:
Please ensure versions get updated in all places for all packages, typically:
constant.ts
package.json
and documents

Reply:
Done.